examples/src/routes/VisualizationByIdentifier.jsx

```diff
@@ -0,0 +1,111 @@
+import VisualizationTableExampleSRC from '!raw-loader!../components/VisualizationTableByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import CustomVisualizationExampleSRC from '!raw-loader!../components/CustomVisualizationExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationBarExampleSRC from '!raw-loader!../components/VisualizationBarByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationLineExampleSRC from '!raw-loader!../components/VisualizationLineByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationAreaExampleSRC from '!raw-loader!../components/VisualizationAreaByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationHeadlineExampleSRC from '!raw-loader!../components/VisualizationHeadlineByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationScatterExampleSRC from '!raw-loader!../components/VisualizationScatterByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationBubbleExampleSRC from '!raw-loader!../components/VisualizationBubbleByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationPieExampleSRC from '!raw-loader!../components/VisualizationPieByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationDonutExampleSRC from '!raw-loader!../components/VisualizationDonutByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationTreemapExampleSRC from '!raw-loader!../components/VisualizationTreemapByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationHeatmapExampleSRC from '!raw-loader!../components/VisualizationHeatmapByIdentifierExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+
+export const VisualizationByIdentifier = () => (
+    <div>
+        <h1>Visualization</h1>
+
+        <p>These are examples of the generic Visualization component.</p>
```

Comment:
I would specify that this test is about URI.
`These are the examples of the generic Visualization component that uses URI to identify insights.`